examples/gke_github/main.tf

```diff
@@ -0,0 +1,144 @@
+terraform {
+  required_version = ">= 0.12.10"
+
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = ">= 4.5.1"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.0.2"
+    }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.10.0"
+    }
+    flux = {
+      source  = "fluxcd/flux"
+      version = ">= 0.0.13"
+    }
+  }
+}
+
+# Flux
+provider "flux" {}
+
+data "flux_install" "main" {
+  target_path = var.target_path
+}
+
+data "flux_sync" "main" {
+  target_path = var.target_path
+  url         = "ssh://git@github.com/${var.organization}/${var.repository_name}.git"
+  branch      = var.branch
+}
+
+# https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/latest/submodules/auth
+module "gke_auth" {
+  source               = "terraform-google-modules/kubernetes-engine/google//modules/auth"
+  project_id           = var.project_id
+  cluster_name         = var.cluster_name
+  location             = var.cluster_region
+  use_private_endpoint = var.use_private_endpoint
+}
+
+provider "kubernetes" {
+  cluster_ca_certificate = module.gke_auth.cluster_ca_certificate
+  host                   = module.gke_auth.host
+  token                  = module.gke_auth.token
+}
+
+provider "kubectl" {
+  cluster_ca_certificate = module.gke_auth.cluster_ca_certificate
+  host                   = module.gke_auth.host
+  token                  = module.gke_auth.token
+  load_config_file       = false
+}
+
+# Kubernetes
+resource "kubernetes_namespace" "flux_system" {
+  metadata {
+    name = var.flux_namespace
+  }
+
+  lifecycle {
+    ignore_changes = [
+      metadata[0].labels,
+    ]
+  }
+}
+
+data "kubectl_file_documents" "install" {
+  content = data.flux_install.main.content
+}
+
+resource "kubectl_manifest" "install" {
+  depends_on = [kubernetes_namespace.flux_system]
+  for_each   = { for v in data.kubectl_file_documents.install.documents : sha1(v) => v }
```

Comment:
Sorry I must have missed this the last time I had a look. You need to align this with the latets Github guide. This is important as it fixes the common issues when upgrading versions.
https://github.com/fluxcd/terraform-provider-flux/blob/3ebd1ec573602e6b6441b0d2b54cdd5e004761b9/examples/github/main.tf#L94

Reply:
Thanks for your review!
Fixed. https://github.com/fluxcd/terraform-provider-flux/pull/112/commits/6260d71768eb3b60cecbf36b6bd1054a588efab1